### PR TITLE
Add "My files don't convert" issue template

### PR DESCRIPTION
**Pull request by leewujung**
This addresses #847.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/my-files-don-t-convert.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+### Error message printouts
+_Required: Copy-paste the entire error messages you encounter here._
+Below is the error messages I received when running the above code:
+```python
+ERROR MESSAGES
+```
+
+### Provide an example file
+_Required: An example file that allow reproducing the problem._
+Here is a link to an example file: SOME_ONLINE_LOCATION
```
Comment (emiliom) [inline .github/ISSUE_TEMPLATE/my-files-don-t-convert.md:35]: Should we place limits on how big the file can be? Say, < 200 MB?

Comment (emiliom) [inline .github/ISSUE_TEMPLATE/my-files-don-t-convert.md:35]:
```suggestion
- Link to an example raw file: SOME_ONLINE_LOCATION
- Raw file sensor model: (FOR EXAMPLE: Simrad EK60)
```

**Comment by emiliom** [comment]
I made a couple of small suggestions. Otherwise, it looks good, thanks!

File: .github/ISSUE_TEMPLATE/my-files-don-t-convert.md (hunk 1 above)
Comment (leewujung) [reply .github/ISSUE_TEMPLATE/my-files-don-t-convert.md]: Yeah that will be good. I'll add that.

**Pull request merged**